scripts/prefix-with-number.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# This can help get patches prefixed by the term of number, like 0001-xxxx.patch, 0002-yyyy.patch, etc.
```

Comment:
can you give a sample invocation for this script, ie how to actually use it?

Reply:
@rn Sure.